container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -276,6 +276,16 @@ if [ ! -f ${TEMPEST_PATH}exclude.txt ] && [ -z ${TEMPEST_EXCLUDE_LIST} ]; then
     touch ${TEMPEST_PATH}exclude.txt
 fi
 
+# This workaround is required for the whitebox-neutron-tempest plugin. We need
+# to be able to specify 600 permissions for the id_ecdsa.
+if [ -f ${HOMEDIR}/id_ecdsa ]; then
+    mkdir ${HOMEDIR}/.ssh
```

Comment:
optional nit: sometimes ssh commands create .ssh, it is possible that in the future it would be created earlier, so "if dir doesn't exist, then create dir" condition can prevent non-zero exit status

Reply:
I do not think we are going to run ssh command in this script but I don't see a reason why we couldn't add  `mkdir -p`:)